tools/wptrunner/wptrunner/browsers/firefox.py

```diff
@@ -119,13 +119,13 @@ def browser_kwargs(logger, test_type, run_info_data, config, **kwargs):
                       "e10s": kwargs["gecko_e10s"],
                       "enable_fission": run_info_data["fission"],
                       "stackfix_dir": kwargs["stackfix_dir"],
-                      "binary_args": kwargs["binary_args"],
+                      "binary_args": kwargs["binary_args"].copy(),
                       "timeout_multiplier": get_timeout_multiplier(test_type,
                                                                    run_info_data,
                                                                    **kwargs),
                       "leak_check": run_info_data["debug"] and (kwargs["leak_check"] is not False),
                       "asan": run_info_data.get("asan"),
-                      "chaos_mode_flags": kwargs["chaos_mode_flags"],
+                      "chaos_mode_flags": kwargs["chaos_mode_flags"].copy(),
```

Comment:
This is going to fail if you run tests locally with `--repeat-until-unexpected` given that the expected kwarg is not present.

Reply:
Actually those `chaos_mode_flags` are of type `int` and there is no need to actually use `copy` here. I'll push an update soon which removes this change, which will make the `--repeat-until-unexpected` argument work again.